runners/runner-local/src/bin.rs

```diff
@@ -1,10 +1,15 @@
 use std::net::SocketAddr;
 
 use apibara_observability::init_opentelemetry;
-use apibara_runner_local::{configuration::Configuration, error::LocalRunnerError, start_server};
+use apibara_runner_local::{
+    configuration::Configuration,
+    error::{LocalRunnerError, LocalRunnerReportExt, LocalRunnerResultExt},
+    start_server,
+};
 use clap::{Args, Parser};
-use error_stack::{Result, ResultExt};
-use tokio_util::sync::CancellationToken;
+use error_stack::Result;
+
+use tokio_util::sync::CancellationToken; // Add missing import
```

Comment:
What missing import?

Reply:
TBH, I don't know, I'll just remote this comment